sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -247,6 +254,37 @@
     @Test
     abstract void analyseSentimentForListLanguageHint();
 
+    // Client builder
```

Comment:
This will require client builder tests to be implemented in both sync and async clients. Instead, can all these tests be done in `TextAnalyticsClientBuilderTest`?